docs/6.coordination/2.procedures.md

```diff
@@ -0,0 +1,249 @@
+Community Users. This is granted to everybody if it is understood that
+the application they aim to developing, is considered a relevant
+reference example for the development of the FIWARE Community.
+
+In order to apply to become a Community User it is necessary to compile
+an application form accessible through the main page of the FIWARE Lab
+portal (<https://account.lab.fiware.org>) - click the *Request Community
+Account Upgrade* - button.
+
+![FIWARE Lab request community account upgrade.](image19.png)
+
+This will open a window in which information about application is
+requested from the user, this information will be used to understand
+exactly what is the planned use of the resources, why they are required
+and in which FIWARE Lab node you plan to use those resources.
+
+![FIWARE Lab community account request](image20.png)
+
+It will create a ticket inside the FIWARE Lab Upgrade Account in Jira to
+be response by the L1 Support team and assigned accordingly to the
+corresponding FIWARE Lab administrator node in order to resolve it.
+
+![FIWARE Lab Upgrade Account (FLUA) Jira project.](./media/image21.png)
+
+From here, the FIWARE Lab node administrators will take those tickets
+and apply the corresponding activities described in section 7.1.3 to
+provide the corresponding resources to the user.
+
+### Help Desk Support
+
+The Help Desk activities are daily part of FIWARE Lab operations. The
+Help Desk is the support that FIWARE Lab experts, Nodes Administrators
+and GEs Owners, give to external and internal users. The Help Desk
+activities are structured in 2 main Level of support in order to
+guarantee that agreed SLAs are achieved.
+
+The first level of support is comprised of a team which is in charge of
+managing all FIWARE Lab incoming tickets. This team is also responsible
+for categorizing all incoming tickets in order to guarantee that Backlog
+statistics (<http://backlog.fiware.org/helpdesk/main>) are always
+properly updated. If is not possible for a member of the first level
+support to solve a problem (e.g. a specific Node failure), he/she will
+assign that ticket to a second level of support consisting of Node
+Administrators and GEs Owners. The Level 2 Help Desk team is in charge
+of providing specific high-level support for the corresponding FIWARE
+Lab Node or FIWARE Generic Enabler.
+
+In the following, we describe in more details the procedures for Level 1
+support.
+
+The Level 1 Help Desk team is the initial contact point for all incoming
+tickets, it provides support for general issues that can be easily
+solved by pointing to the FAQ, Stack Overflow, groups or other
+documentation. Moreover, its responsibility is to filter, categorise and
+forward to Level 2 support (Node Owners or GE Owners) all those tickets
+that it is not able to answer. It is also responsible for managing
+Community Account requests.
+
+The Level 1 Help Desk team is organized in 8x5 (Mon to Fri) scheduled
```

Comment:
The Level 1 Help Desk team is organized in 8x5 (Monday to Friday) scheduled
shifts from 08:00 to 17:00. The team is composed by people from
FIWARE Foundation.

Reply:
There are now 2 paragraphs. Please delete the next one:

"The Level 1 Help Desk team is organized in 8x5 (Mon to Fri) scheduled shifts from 08:00 to 17:00. The team is composed by people from Engineering, FBK, FIWARE Foundation, Martel that are on duty on a respective week according to the calendar available at: https://docs.google.com/spreadsheets/d/1X9iLY9Znd3Rh-GM4qffGEHRNKqKcHZqdm-dP0rq8OVo/edit#gid=1383004526. Regarding INFOTEC and ITSM they are in charge of the management of the FIWARE Lab Mexico node, providing support to the corresponding Level 1 Help Desk team."